mars/web/tests/test_api.py

```diff
@@ -204,6 +205,21 @@ def setUp(self):
 
         self.start_web(scheduler_address)
 
+    def tearDown(self):
+        self._web.stop()
+        self._pool.stop()
+
+    def start_web(self, scheduler_address):
+        import gevent.monkey
```

Comment:
The travis test test_api with process, so the monkey patch will not effect other tests, but in appveyor, we didn't run in process for this particular test. Please make sure this monkey patch will not effect other ut for now and the coming future.

Reply:
Now use mock server to test api, so the problem may be ignored.